dist/nix/vimrc-with-plugins.nix

```
@@ -0,0 +1,28 @@
+{ pkgs ? import <nixpkgs>{}
+, additionalPlugins? []
+, additionalCustPlugins? {}
+, additionalVimrc? ""#(pkgs.builtins.readFile ./vimrc)
```

Remove debug comment?
```suggestion
, additionalVimrc? ""
```

---

it meant to be an example line if you want to add your own vimrc configuration. you could use this function either passing your vimrc this way or change this default value and copy paste it.